Robot_Adapter/Convert/ToRobot/Loads/BarVaryingDistributedLoad.cs

```diff
@@ -36,51 +36,42 @@ public static partial class Convert
         public static void ToRobot(this BarVaryingDistributedLoad load, RobotSimpleCase sCase, RobotGroupServer rGroupServer)
         {
             //Replace null vectors with empty vector
-            load.ForceA = load.ForceA ?? new oM.Geometry.Vector();
-            load.ForceB = load.ForceB ?? new oM.Geometry.Vector();
-            load.MomentA = load.MomentA ?? new oM.Geometry.Vector();
-            load.MomentB = load.MomentB ?? new oM.Geometry.Vector();
+            load.ForceAtStart = load.ForceAtStart ?? new oM.Geometry.Vector();
+            load.ForceAtEnd = load.ForceAtEnd ?? new oM.Geometry.Vector();
+            load.MomentAtStart = load.MomentAtStart ?? new oM.Geometry.Vector();
+            load.MomentAtEnd = load.MomentAtEnd ?? new oM.Geometry.Vector();
 
-            if (load.ForceA.Length() == 0 && load.ForceB.Length() == 0)
+            if (load.ForceAtStart.Length() == 0 && load.ForceAtEnd.Length() == 0)
             {
                 Engine.Reflection.Compute.RecordWarning("Zero distributed forces are not pushed to Robot");
                 return;
             }
 
-            if (load.MomentA.Length() != 0 && load.MomentB.Length() != 0)
+            if (load.MomentAtStart.Length() != 0 && load.MomentAtEnd.Length() != 0)
```

Comment:
```suggestion
            if (load.MomentAtStart.Length() != 0 || load.MomentAtEnd.Length() != 0)
```

You could have the case where the moment goes from Mx =0 kNm to Mx = +5 kNm